packages/usa-modal/src/test/modal.spec.js

```diff
@@ -25,22 +28,30 @@ tests.forEach(({name, selector: containerSelector}) => {
     let openButton1;
     let openButton2;
     let overlay;
+    let comboList;
+    let CBToggle;
```

Comment:
Let's use camel casing and use a more descriptive name.

Reply:
Resolved in https://github.com/uswds/uswds/pull/5049/commits/577d1b94baf2d62a48859bce2fd3e3424df0c8cd